tests/dragonfly/connection_test.py

```diff
@@ -1,7 +1,24 @@
+import pytest
 import asyncio
 import aioredis
 import async_timeout
-#from conftest import DATABASE_INDEX
+
+'''
+Test the monitor command.
+Open connection which is used for monitoring
+Then send on other connection commands to dragonfly instance
+Make sure that we are getting the commands in the monitor context
+'''
+
+
+@pytest.mark.asyncio
+async def test_monitor_command(async_pool):
+    def generate(max):
+        for i in range(max):
+            yield f"key{i}", f"value={i}"
+
+    messages = {a: b for a, b in generate(5)}
+    assert await run_monitor(messages, async_pool)
```

Comment:
I replaced all the manual loop code with pytest_asyncio

I also moved the functions a little so that the comments are directly above the main testing function (more readable)